common/src/main/templates/io/netty/util/collection/KObjectMap.template

```diff
@@ -63,11 +63,6 @@ public interface @K@ObjectMap<V> {
     V put(@k@ key, V value);
 
     /**
-     * Puts all of the entries from the given map into this map.
-     */
-    void putAll(@K@ObjectMap<V> sourceMap);
```

Comment:
@nmittler - Maybe you haven't pushed the code yet? I'm still seeing this:

```java
    @Override
    public void putAll(Map<? extends @O@, ? extends V> sourceMap) {
        if (sourceMap instanceof @K@ObjectHashMap) {
            // Optimization - iterate through the arrays.
            @K@ObjectHashMap<V> source = (@K@ObjectHashMap<V>) sourceMap;
            for (int i = 0; i < source.values.length; ++i) {
                V sourceValue = source.values[i];
                if (sourceValue != null) {
                    put(source.keys[i], sourceValue);
                }
            }
            return;
        }

        // Otherwise, just add each entry.
        for (Entry<? extends @O@, ? extends V> entry : sourceMap.entrySet()) {
            put(entry.getKey(), entry.getValue());
        }
    }
```